Content.Server/Atmos/EntitySystems/AirtightSystem.cs

```diff
@@ -44,7 +44,15 @@ private void OnAirtightInit(EntityUid uid, AirtightComponent airtight, Component
 
         private void OnAirtightShutdown(EntityUid uid, AirtightComponent airtight, ComponentShutdown args)
         {
-            SetAirblocked(airtight, false);
+            var xform = EntityManager.GetComponent<TransformComponent>(uid);
```

Comment:
```suggestion
            var xform = Transform(uid);
```

Reply:
I'm a fucking idiot, I even did it in my other PR, I EVEN DID IT IN THIS PR